Src/DependencyCollector/Shared/DependencyTrackingTelemetryModule.cs

```diff
@@ -189,6 +201,17 @@ protected virtual void Dispose(bool disposing)
                     {
                         this.sqlEventListener.Dispose();
                     }
+
+                    if (this.httpDesktopDiagnosticSourceListener != null)
```

Comment:
It looks like it was disposed above unless there is a slight difference in the variable name I didn't spot.